<launch>

	<node pkg="nodelet" type="nodelet" name="nodelet_manager"  args="manager"/>
	
	<node pkg="nodelet" type="nodelet" name="collision_avoidance_nodelet"
		args="load collision_avoidance/CA nodelet_manager" > <!-- output="screen"> -->
		<rosparam file="$(find collision_avoidance)/config/collision_avoidance_params.yaml" command="load" />
	</node>
	
	

	
	<arg name="event_num" default="16" />

    <node name="joy_rumble_node" pkg="joy_rumble" type="joy_rumble_node" output="screen" >
    	<param name="event" type="str" value="/dev/input/event$(arg event_num)" />
    </node>
	
</launch>
